changelog.d/11985.misc

```diff
@@ -0,0 +1 @@
+Use a bot User-Agent for URL preview queries.
```

Comment:
Thinking through this more, I think we should make this a `.feature` file which says "Fetch images when previewing Twitter URLs."

You can also credit yourself if you would like "Contributed by (your name or @AndrewRyanChama)."